nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,21 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    # to control floating point error problems
+    # during given input value "n_cuts"
+    epsilon = np.finfo(np.float32).eps
+    difference = abs(round(n_cuts) - n_cuts)
+    message = ("The number of cuts in the given direction %s must be "
+               "an integer which should be greater than 0 and "
+               "less than or equal to %d. You provided n_cuts=%s " % (
+                   direction, this_shape, n_cuts))
+    if not isinstance(n_cuts, numbers.Number) or n_cuts < epsilon:
```

Comment:
Not sure what you mean. What happened to the condition about n_cuts being close to an int ?